# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/workers/MessageEventRunnable.cpp
# Commit: 7802e5f1bd13
# Full Hash: 7802e5f1bd1395f19f2d78636787c41f64192bd7
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2024-04-19 15:22:44
# Regressor Bug: 1769913
# File Overlap Count: 3
# Description:
#   Bug 1769913 - P3 Remove WorkerRunnable::mWorkerPrivate. r=dom-worker-reviewers,asuth
#   
#   WorkerRunnable no longer keeps a raw pointer(mWorkerPrivate) for the associated WorkerPrivate in this patch.
#   Removing the WorkerRunnable::mWorkerPrivate needs to fix the following problems.
#   
# ==============================================================================

diff -r ffc69aa8f2c3 -r 7802e5f1bd13 dom/workers/MessageEventRunnable.cpp
--- a/dom/workers/MessageEventRunnable.cpp	Fri Apr 19 09:41:57 2024 +0000
+++ b/dom/workers/MessageEventRunnable.cpp	Fri Apr 19 09:41:58 2024 +0000
@@ -15,7 +15,7 @@
 namespace mozilla::dom {
 
 MessageEventRunnable::MessageEventRunnable(WorkerPrivate* aWorkerPrivate)
-    : WorkerDebuggeeRunnable(aWorkerPrivate, "MessageEventRunnable"),
+    : WorkerDebuggeeRunnable("MessageEventRunnable"),
       StructuredCloneHolder(CloningSupported, TransferringSupported,
                             StructuredCloneScope::SameProcess) {}
 
@@ -111,8 +111,7 @@
 
 MessageEventToParentRunnable::MessageEventToParentRunnable(
     WorkerPrivate* aWorkerPrivate)
-    : WorkerParentDebuggeeRunnable(aWorkerPrivate,
-                                   "MessageEventToParentRunnable"),
+    : WorkerParentDebuggeeRunnable("MessageEventToParentRunnable"),
       StructuredCloneHolder(CloningSupported, TransferringSupported,
                             StructuredCloneScope::SameProcess) {}
 